tests/handlers/RodParts.test.js

```diff
@@ -0,0 +1,55 @@
+const RodPartsHandler = require('../../calculators/handlers/RodParts');
+const { APPLICATION_WORTH } = require('../../constants/applicationWorth');
+const BaseHandlerTest = require('./BaseHandlerTest');
+
+const testCases = [
+    {
+        description: 'Applies correctly',
+        item: {
+            itemId: 'ROD_OF_THE_SEA',
+            extraAttributes: {
+                line: { part: 'titan_line', donated_museum: 1 },
```

Comment:
Rod parts sometimes have the `donated_museum` key on them which is set to `1` (true). We should probably count this as unsoulbound networth. Not sure the best way to go about this...